example/Avoidable.js

```diff
@@ -0,0 +1,201 @@
+import React from 'react';
```

Comment:
This file will be removed after the lib is deployed to npm, right?

Reply:
Yes. This file is copied over until we can use it from npm